src/test/java/github/weichware10/analyse/DiagrammTest.java

```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.